spring-batch-docs/asciidoc/readersAndWriters.adoc

```diff
@@ -1777,6 +1791,9 @@ as with any `ItemReader`, adding extra input (in this case a file) could cause p
 issues when restarting. It is recommended that batch jobs work with their own individual
 directories until completed successfully.
 
+NOTE: Input resources are ordered using `MultiResourceItemReader#setComparator(Comparator)`
```

Comment:
I guess this note is important to add here (I took it from the javadoc of `MultiResourceItemReader `)

Reply:
"ordered using" should be "ordered by using". (It's a usage that is passing from being common, but some of our audience will notice.)